JarrydWorland/2021_Capstone_GP2_Team
# Item homing shot

## Description
Adds a new item that adds a homing effect to the players projectiles, the effect gets stronger as the item stacks.

## How to test
Play and ensure the homes correctly on both mouse and controller. Ensure the item effect goes away when the item is dropped.
